admin/cli/cmd/base/options.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2025 Microsoft Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package base
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/spf13/cobra"
+
+	"github.com/Azure/ARO-HCP/admin/cli/pkg/cli/auth"
+	"github.com/Azure/ARO-HCP/admin/cli/pkg/cli/request"
+)
+
+func DefaultAuthOptions() *RawAuthOptions {
+	return &RawAuthOptions{}
+}
+
+type RawAuthOptions struct {
+	TenantID          string
+	ClientID          string
+	Scopes            []string
+	KeyVaultURI       string
+	CertificateSecret string
+	Endpoint          string
+	Host              string
+	Insecure          bool
+}
+
+func (o *RawAuthOptions) BindFlags(cmd *cobra.Command) error {
+	cmd.Flags().StringVar(&o.TenantID, "ga-auth-tenant-id", o.TenantID, "Tenant ID to use for the Geneva Action auth certificate")
```

Comment:
Perhaps my misunderstanding, but what does it mean to use a tenant ID for a certificate? Or scopes? Is this the tenant in which we use the cert to auth, and are those the OAuth scopes we ask for?